lib/runners/julia.js

```diff
@@ -12,7 +12,7 @@ module.exports.run = function run(opts, cb) {
       if (opts.setup) codeWriteSync('julia', opts.setup, juliaCodeDir);
       runCode({
         name: 'julia',
-        args: ['-P', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
+        args: ['', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
```

Comment:
I read some docs and I see that `-P` was deprecated.
By removing that flag, the first argument that `julia` receives is now an expression ("'push!'"). I think this is the cause of the empty output issue.